mdata/chunk/format.go

```diff
@@ -10,4 +10,5 @@ type Format uint8
 // identifier of message format
 const (
 	FormatStandardGoTsz Format = iota
+	FormatWithLen
```

Comment:
Actually, we pretty much use the term `span` everywhere to convey the timerange covered in seconds
For consistency let's adopt the term span also here instead of len and size.